pkg/config/runtime_config.go

```diff
@@ -40,10 +42,21 @@ const (
 	defaultQPS = 1e6
 	// High enough Burst to fit all expected use cases. Burst=0 is not set here, because
 	// client code is overriding it.
-	defaultBurst           = 1e6
-	defaultWebhookCertDir  = ""
-	defaultWebhookCertName = ""
-	defaultWebhookKeyName  = ""
+	defaultBurst                = 1e6
+	defaultWebhookCertDir       = ""
+	defaultWebhookCertName      = ""
+	defaultWebhookKeyName       = ""
+	defaultWebhookTLSMinVersion = "1.3"
+)
+
+var (
+	supportedTLSVersions = map[string]struct{}{
+		"":    {},
+		"1.0": {},
+		"1.1": {},
```

Comment:
We definitely shouldn't support 1.0 or 1.1

Reply:
Yeah, these ones are too outdated for now. I added them since the controller-runtime still supports them. I'm fine to remove them along with `""` as it turns into `1.0`.

Reply:
`1.0`, `1.1` removed.